plugins/node/opentelemetry-instrumentation-dataloader/src/instrumentation.ts

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  InstrumentationBase,
+  InstrumentationNodeModuleDefinition,
+  isWrapped,
+} from '@opentelemetry/instrumentation';
+import {
+  diag,
+  trace,
+  context,
+  Link,
+  SpanStatusCode,
+  SpanKind,
+} from '@opentelemetry/api';
+import { DataloaderInstrumentationConfig } from './types';
+import { VERSION } from './version';
+import * as Dataloader from 'dataloader';
```

Comment:
```suggestion
import type * as Dataloader from 'dataloader';
```
An instrumentation should not runtime depend on the module it instruments. dev-depending on types is fine.

Reply:
Good catch!